wiki/fr/fr-Software-Manual.md

```diff
@@ -0,0 +1,226 @@
+
+Fenêtre Principale
+------------------
+
+![Fenêtre Principale](https://user-images.githubusercontent.com/20726856/97360410-77476880-189e-11eb-99c6-f4a68258bf31.png)
+
+### LEDs d'état
+
+La LED **Délai** indique l'état actuel du retard audio :
+
+* **Vert** - Le délai est parfait pour une bonne jam session
+
+![image](https://user-images.githubusercontent.com/4561747/99384024-b6c60b00-28c6-11eb-9e7d-aa1e4245353f.png)
+
+* **Jaune** - Une session est toujours possible mais elle peut être plus difficile à jouer
+
+![image](https://user-images.githubusercontent.com/4561747/99384041-bb8abf00-28c6-11eb-96ab-c1c08acf2a85.png)
+
+* **Rouge** - Le délai est trop important pour jouer
+
+![image](https://user-images.githubusercontent.com/4561747/99384044-bded1900-28c6-11eb-8971-9ab6ee1908b5.png)
+
+
+**Tampons** indique l'état actuel de l'audio/du streaming. Si le voyant est **rouge**, le flux audio est interrompu. Cela peut être dû à l'un des problèmes suivants :
+
+- Le tampon de gigue[^1] réseau n'est pas assez grand pour la gigue actuelle de l'interface réseau/audio.
+- Le délai de l'interface audio (taille de la mémoire tampon) est trop faible (voir la fenêtre des paramètres).
+- Le debit montant ou descendant est trop élevé pour votre bande passante Internet.
+- Le processeur du client ou du serveur est à 100 %.
+
+### Niveau d'entrée
+
+![Niveau d'entrée](https://user-images.githubusercontent.com/20726856/97363247-9516cc80-18a2-11eb-8195-1e887ee8040f.png)
+
+Ceci indique le niveau des deux canaux stéréo pour votre entrée audio.
+Veillez à ne pas écrêter le signal d'entrée pour éviter la saturation du signal audio (les LEDs indiquent l'écrêtage lorsqu'il se produit).
+
+### Bouton _Tchate_
+
+![Tchate](https://user-images.githubusercontent.com/20726856/97360717-d7d6a580-189e-11eb-84db-80536e17dc30.png)
+
+Le texte saisi dans cette fenêtre est envoyé à tous les clients connectés.
+Si un nouveau message de tchate arrive et que la fenêtre du tchate n'est pas encore ouverte, elle est ouverte automatiquement pour tous les clients.
+
+### Profil de musicien
+
+![Profil de musicien](https://user-images.githubusercontent.com/20726856/97360811-f9379180-189e-11eb-98bf-7f44f884adaa.png)
+
+Dans le menu Vue, sélectionnez Mon profil pour définir votre alias/nom qui s'affiche sous votre canal dans la console de mixage audio du serveur. Si un instrument et/ou un pays est défini, les icônes correspondant à ces choix seront également affichées sous votre potentiomètre. Le paramètre de compétence modifie la couleur de fond du de l'etiquette du potentiomètre et l'entrée de la ville apparaît dans l'info-bulle au survol de l'etiquette du canal (voir la capture d'écran ci-dessous).
+
+![Astuce pour les etiquettes de canal](https://user-images.githubusercontent.com/20726856/97360889-166c6000-189f-11eb-802d-330e761ed017.png)
```

Comment:
"chariot" -> "potentiomètre" (already used in this doc)
"chariot" is not ususally used (by "us", the "sound mans" : « sondiers ») to describe a mixing channel or a "console slice" (« tranche de console »).
"Astuce" -> "Info-bulle" (because alt content, we should describe the image)

-> "Info bulle de l'étiquette du potentiomètre"